plugins/flutter_plugins/url_launcher_fde/linux/url_launcher_plugin.cc

```diff
@@ -44,6 +44,11 @@ class UrlLauncherPlugin : public flutter::Plugin {
       std::unique_ptr<flutter::MethodResult<EncodableValue>> result);
 };
 
+// utility function to compare url schemes
+static bool startsWith(const std::string& s, const std::string& prefix) {
```

Comment:
Nits:
- No need for `static` since it's in an anonymous namespace.
- First letter should be uppercase per C++ style guide.
- Please move up to before the class declaration, so it's not sandwiched between the class declaration and definitions.